### DISCUSSION TITLE
RPi Camera stream over network to Motion

### DISCUSSION THREAD

**Discussion by spindles7** [Motion-Project/motion#1690] [Q&A]
Have got a Raspberry Pi with the RPi Camera V.2 which I have just installed RaspberryPi OS Bullseye from an image.   Trying to use the built-in libcamera-vid command to stream the video to another machine (running Debian Bookworm and Motion 4.5.1).   I can view the stream using VLC on my Windows machine with the string:

tcp/h264://192.168.2.254:3000

where 192.168.2.254 is the IP address of the Raspberry Pi and 3000 is the port specified in the libcamera-vid command which is:

libcamera-vid -t 0 --inline --listen -o tcp://0.0.0.0:3000

Have tried various netcam_url settings, but none of the will work.    Anybody any clues as to what the correct settings in motion.conf could be to allow the stream to be captured coirrectly?

Thanks,

spindles7

**Comment by lolorc** [comment]
ever looked at mediamtx or go2rtc ? I'm using mediamtx to stream off my rpis for instance, that's also doable with go2rtc as well.
